strings/books/bee_book.txt

```diff
@@ -0,0 +1,35 @@
+<p>There are rumors of other methods of hatching Bees, such as sneezing them out. While very silly, there may be some truth to these, so keep an ear out for interesting rumors!</p>
+<h1 id="chapter-two-types-of-bees">Chapter Two: Types of Bees</h1>
+<h2 id="the-greater-domestic-space-bee">The Greater Domestic Space-Bee</h2>
+<p>The most common of Bees in outer space, and the one from which all potential springs! You're going to see a lot of these, and that's a very good thing!</p>
+<h2 id="colorful-domestic-space-bee">Colorful Domestic Space-Bee</h2>
+<p>Sometimes, especially with unorthodox hatchings, Bees will take on the color of the human who hatched them. Specifically, the color of their hair. It's fascinating, but these Bees otherwise act like any other Domestic Space-Bee.</p>
+<h2 id="guardbud-bee">Guardbud-Bee</h2>
+<p>These strange Bees are somehow infused with DNA from a PR-6 Robuddy. No one quite knows how it happens. The eggs for these are kept out of the mainstream distribution channels, but there may be someone who can help you get your hands on them...</p>
+<h2 id="queen-space-bee">Queen Space-Bee</h2>
+<p>These come in regular, colorful, and Guardbud-Bee varieties. Sometimes they're a little bigger than average, and occasionally they even have a crown! You can get these by feeding larva Royal Jelly.</p>
+<h2 id="fat-and-sassy-space-bee">Fat and Sassy Space-Bee</h2>
+<p>This is a fat, sassy space Bee. Don't make fun of their weight - their sass is downright painful if they turn it on you. They tend to really like bubble baths.</p>
+<h2 id="greater-domestic-sea-bee">Greater Domestic Sea-Bee</h2>
+<p>These Bees are found in aquatic locales, and have cute little goggles! Adorable!</p>
+<h2 id="psychadelic-space-bee">Psychadelic Space-Bee</h2>
+<p>Woah... these Bees are like, far out man... their eyes flicker and when they hug you it buzzes real good. Exposure to certain substances as an egg or larva causes Bees to become this when they grow up.</p>
+<h2 id="chromatic-space-bee">Chromatic Space-Bee</h2>
+<p>These Bees change between different colors constantly! Very popular among hardcore gamers. Basically never seen without Psychadelic Space-Bees in the area.</p>
+<h2 id="zombees">Zombees</h2>
+<p>These Bees look unwell... but they'll produce honey for you just the same, if you give them lots of love!</p>
+<h1 id="chapter-three-specific-bees">Chapter Three: Specific Bees</h1>
+<h2 id="heisenbee">Heisenbee</h2>
+<p>The beloved pet of the Research Director of NanoTrasen Space Station 13. Tends to wear extremely stylish hats.</p>
+<h2 id="bombini">Bombini</h2>
+<p>This li'l guy deals in all sorts of Bee-related goods. Definitely someone you want to Bee-friend!</p>
```

Comment:
Minor nitpick, Bombini is a she. Overall I enjoyed reading the book, very cute and informative

Reply:
Oh, huh, I never really thought about it, but that makes sense. I'll fix it right away.